pyuvdata/uvdata/mir_parser.py

```diff
@@ -1761,7 +2108,13 @@ def _fix_acdata(self):
 
     def read(
         self,
-        filepath,
+        filepath=None,
+        *,
+        auto_defaults=True,
+        compass_soln=None,
+        make_v3_compliant=None,
+        old_vis_format=None,
```

Comment:
Same here?

Reply:
See comment above -- using the None to allow for dynamic default behavior depending on file version.